nixnet/_session/frames.py

```diff
@@ -22,6 +22,23 @@ def __repr__(self):
     def _create_item(self, handle, index, name):
         return Frame(handle, index, name)
 
+    @property
+    def payld_len_max(self):
+        """int: Returns the maximum payload length of all frames in this session, expressed as bytes (0-254).
+
+        For CAN Stream (Input and Output), this property depends on the XNET
+        Cluster CAN I/O Mode property. If the I/O mode is CAN, this property is
```

Comment:
Looks inconsistent in referring to the enum values, so `CAN` -> `constants.CaNioMode.CAN`?